docs/networks/arbitrum-one.md

```diff
@@ -58,16 +62,25 @@ option can be used.
-| --------------------------------------------- | ------------------------------- | ---------------------------------------------------------------------------------------- |
-| `INDEXER_SERVICE_ETHEREUM`                    | `--ethereum`                    | An Arbitrum mainnet node/provider                                                        |
-| `INDEXER_SERVICE_INDEXER_ADDRESS`             | `--indexer-address`             | Ethereum address of mainnet indexer                                                      |
-| `INDEXER_SERVICE_MNEMONIC`                    | `--mnemonic`                    | Ethereum mnemonic for mainnet operator                                                   |
-| `INDEXER_SERVICE_NETWORK_SUBGRAPH_DEPLOYMENT` | `--network-subgraph-deployment` | `QmSWxvd8SaQK6qZKJ7xtfxCCGoRzGnoi2WNzmJYYJW9BXY`                                         |
-| `INDEXER_SERVICE_NETWORK_SUBGRAPH_ENDPOINT`   | `--network-subgraph-endpoint`   | `https://gateway-arbitrum.network.thegraph.com/api/[api-key]/subgraphs/id/DZz4kDTdmzWLWsV373w2bSmoar3umKKH9y82SUKr5qmp`           |
-| `INDEXER_SERVICE_CLIENT_SIGNER_ADDRESS`       | `--client-signer-address`       | `0xc483960d4D58eabc434Dc88a620AdFd883D6Dd4e`                                             |
+### Indexer Service rs *and* TAP Agent
+
+They are configured using a TOML file provided with `--config`. You should start with [the minimal config example](https://github.com/graphprotocol/indexer-rs/blob/main/config/minimal-config-example.toml). You can find the full config [here](https://github.com/graphprotocol/indexer-rs/blob/main/config/maximal-config-example.toml) and the default values 
+[here](https://github.com/graphprotocol/indexer-rs/blob/main/config/default_values.toml).
+
+| Variable                          | Value                                                                                                                   |
+| --------------------------------- | ----------------------------------------------------------------------------------------------------------------------- |
+| `indexer.indexer_address`         | Ethereum address of testnet indexer                                                                                     |
+| `indexer.operator_mnemonic`       | Ethereum mnemonic for testnet operator                                                                                  |
+| `subgraphs.network.deployment_id` | `QmSWxvd8SaQK6qZKJ7xtfxCCGoRzGnoi2WNzmJYYJW9BXY`                                                                        |
+| `subgraphs.network.query_url`     | `https://gateway-arbitrum.network.thegraph.com/api/[api-key]/subgraphs/id/DZz4kDTdmzWLWsV373w2bSmoar3umKKH9y82SUKr5qmp` |
+| `subgraphs.escrow.deployment_id`  | `QmUhiH6Z5xo6o3GNzsSvqpGKLmCt6w5WzKQ1yHk6C8AA8S`                                                                        |
```

Comment:
@aasseman is it still the case that user is expected to provide either `query_url` OR `deployment_id` for `subgraphs.network`and `subgraphs.escrow`? If it is we may want to make that clear here as well